internal/store/store.go

```diff
@@ -51,8 +51,16 @@ func NewStore(reducer Reducer, logActions LogActionsFlag) *Store {
 	}
 }
 
-func NewStoreForTesting() *Store {
-	return NewStore(EmptyReducer, false)
+func NewStoreForTesting() (*Store, <-chan Action) {
```

Comment:
i'm not sure that an Action channel is the right API here. (the panic() seems like a good sign that the api is wrong)

it might eventually make more sense to use a MockStore with a getActions() method, like this project: https://github.com/dmitry-zaets/redux-mock-store

Reply:
kinda feeling https://www.jtolio.com/2016/03/go-channels-are-bad-and-you-should-feel-bad/ right now :) (and how often I've wanted growing buffered channels)

I've changed it to use a `func getActions() []Action`, but I'm not sure it feels better.